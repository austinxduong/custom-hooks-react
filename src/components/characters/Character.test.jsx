import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

import Avatar from './Character';

describe('Avatar component aka Character.jsx', () => {
  it('renders individual character from Avatar component', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Avatar
          name= "idk"
          allies= "animal"

        />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
