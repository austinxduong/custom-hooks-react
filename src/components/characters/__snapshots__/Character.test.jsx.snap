// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component aka Character.jsx renders individual character from Avatar component 1`] = `
<DocumentFragment>
  <figure>
    <img
      alt="idk"
    />
    <figcaption>
      <p>
        Name: idk
      </p>
      <p>
        Allies: animal
      </p>
    </figcaption>
  </figure>
</DocumentFragment>
`;
